easybuild/easyconfigs/t/Triton/Triton-3.5.0-gfbf-2025b-CUDA-12.9.1.eb

```
@@ -0,0 +1,138 @@
+easyblock = 'PythonBundle'
+
+name = 'Triton'
+version = '3.5.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://triton-lang.org/'
+description = """Triton is a language and compiler for parallel programming. It aims to provide a
+Python-based programming environment for productively writing custom DNN compute
+kernels capable of running at maximal throughput on modern GPU hardware."""
+
+toolchain = {'name': 'gfbf', 'version': '2025b'}
+
+builddependencies = [
+    ('CMake', '3.31.8'),
+    ('Ninja', '1.13.0'),
+    ('nlohmann_json', '3.12.0'),
+    ('git', '2.50.1'),
+    ('libxml2', '2.14.3'),
+    ('lit', '18.1.8'),
+    ('ncurses', '6.5'),
+]
+dependencies = [
+    ('CUDA', '12.9.1', '', SYSTEM),
+    ('Python', '3.13.5'),
+    ('Python-bundle-PyPI', '2025.07'),
+    ('SciPy-bundle', '2025.07'),
+    ('pybind11', '3.0.0'),
+    ('Z3', '4.15.1'),
+    ('zstd', '1.5.7'),
+    ('zlib', '1.3.1'),
+]
+
+local_arch = {"arm64": "sbsa", "aarch64": "sbsa"}.get(ARCH, ARCH)
+
+components = [
+    # Hash from cmake/llvm-hash.txt
+    ('LLVM', '7d5de3033187c8a3bb4d2e322f5462cdaf49808f', {
+        'easyblock': 'CMakeNinja',
+        'source_urls': ['https://github.com/llvm/llvm-project/archive/'],
```

```suggestion
        'source_urls': ['https://github.com/triton-lang/llvm-project/archive/'],
```

This commit _does not_ exist in upstream LLVM (well, the PR referenced in the commit seemingly does, but not the commit hash)